ivy/functional/backends/tensorflow/linear_algebra.py

```diff
@@ -576,7 +574,25 @@ def svdvals(
     return ret
 
 
-@with_unsupported_dtypes({"2.13.0 and below": ("complex",)}, backend_version)
+@with_unsupported_dtypes(
+    {
+        "2.13.0 and below": (
+            "complex",
+            "float16",
+            "float64",
+            "bfloat16",
+            "int8",
+            "int16",
+            "int32",
+            "int64",
+            "uint8",
+            "uint16",
+            "uint32",
+            "uint64",
+        )
+    },
+    backend_version,
+)
```

Comment:
It'll be much more simple if you just add `float32` and `float64` in the supported dtypes decorator.